autoload/lightline/colorscheme/simpleblack.vim

```diff
@@ -0,0 +1,43 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/simpleblack.vim
+" Author: lucasprag
+" License: MIT License
+" Last Change: 2019/10/28 22:54:01.
+" =============================================================================
+let s:black = [ '#000000', '0' ]
+let s:black2 = [ '#121212', '233' ]
+
+let s:gray = [ '#262626', '235' ]
+let s:gray2 = [ '#3a3a3a', '237' ]
+let s:gray3 = [ '#4e4e4e', '239' ]
+let s:gray4 = [ '#626262', '241' ]
+
+let s:violet = [ '#CF73E6', '170' ]
+
+let s:blue = [ '#5f87af', '67' ]
+let s:blue2 = [ '#91AADF', '110' ]
+
+let s:green = [ '#57BA37', '71' ]
+let s:gold = [ '#F0D50C', '220' ]
```

Comment:
Can you make the hex consistently lowercase?

Reply:
Sure, thanks for reviewing =)

Reply:
Sure, thanks for reviewing @itchyny =)